engine/src/main/java/org/terasology/recording/RecordAndReplayUtils.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.recording;
+
+/**
+ * Saves data important for Record and Replay, such as the RecordAndReplayStatus, the title of the current game, if a
+ * shutdown was requested, and the amount of files used to serialize RecordedEvents.
+ */
+public final class RecordAndReplayUtils {
+    private static RecordAndReplayStatus recordAndReplayStatus = RecordAndReplayStatus.NOT_ACTIVATED;
+    private static String gameTitle;
+    private static boolean shutdownRequested;
+    private static int fileCount = 1;
+    private static int fileAmount;
```

Comment:
Util class with a static state looks strange ;) I suspect this is the state for the event replay. Should probably be an object and an instance of at least the recording event system. This may take some extra architecture work though to find the right spot for instance creation and passing.